src/lib/monitoring/alarmDescriptions.ts

```diff
@@ -0,0 +1,62 @@
+export const defaultAlarmDescriptionTemplate = `
+Consider the following:
+- Be precise: character limit of 1024 for the alert description
+- Actionability: Is it actionable, is the alert even needed?
+- Prefer examples: share e.g. CloudWatch Insights queries
+- Reusability: If instruction is generic open a PR to mca-cli :)
+`.trim();
+
+const lambdaErrorsAlarmDescriptionTemplate = `
+- Evaluate the criticality of alert:
+  * Check the amount of errors
+  * If there are a lot of errors inform the product owner immediately
+- Find the requestId of the error with CloudWatch Insights query:
+  fields @timestamp, @message
+  | sort @timestamp desc
+  | filter @message like /ERROR/
+- Get the logs for the requestId:
+  fields @timestamp, @message
+  | sort @timestamp desc
+  | filter @requestId = "requestIdHere"
+- Check if a development ticket exists of this issue
+  * If not create one
+`.trim();
+
+const lambdaDurationAlarmDescriptionTemplate = `
+- Check metric history for changes to durations
+- Evaluate whether alarm threshold or applications needs to change
+- CloudWatch Insights query to find offending durations:
+  fields @timestamp, @message
+  | sort @timestamp desc
+  | filter @duration > durationThresholdHere
+`.trim();
+
+const lambdaInvocationsAlarmDescriptionTemplate = `
+- Check metric history for changes to invocations
+- Evaluate whether alarm threshold or applications needs to change
+- CloudWatch Insights query to check the invocation counts:
+  fields @timestamp, @message
+  | sort @timestamp desc
+  | filter @message like /START RequestId:/
+  | stats count() by bin(5m)
+`.trim();
+
+const lambdaThrottlesAlarmDescriptionTemplate = `
+- Check metric history for throttles
+- Evaluate the severity
+  * Check how retry logic has been implemented
+  -> if retry logic is missing, issue is CRITICAL
+  * If situation is critical and urgent, request
+    concurrency limit extension from AWS support immediately
+  * Check whether the issue affects data integrity
+  * Add retry logic if it's missing!
+`.trim();
+
+export default {
```

Comment:
I believe it would be better to export all separately and nothing as default. Then you could be using it better in the other parts with `import * as descriptions from './descriptions'`

Having one exported separately and others as default is quite confusing

Reply:
fixed it, thoughts?

Reply:
So now it would be `default.default` export for the default template?

Reply:
just import only default

Reply:
so it would be descriptions.default

Reply:
I still don't really like it and would prefer everything to be exported separately to allow importing only one part if needed as now I would need to include all when I just need the `defaultAlarmDescriptionTemplate` for example. If I needed all I could use `import * as descriptions ...`

Reply:
I will approve these changes as there is nothing really issue other than my personal opinion regarding the exports :)

Reply:
default export isn't definitely a good fit for everything, but I like to use it in this kind of data structure. But, I'll change it to separate exports, since it seems to be a convention in this repo. We should have some sort of coding conventions readme for these, so we could always refeer to it. 👍